src/rules/noRequireImportsRule.ts

```diff
@@ -26,27 +26,20 @@ export class Rule extends Lint.Rules.AbstractRule {
 }
 
 class NoRequireImportsWalker extends Lint.RuleWalker {
-    public visitVariableStatement(node: ts.VariableStatement) {
-        const declarations = node.declarationList.declarations;
-        for (let decl of declarations) {
-            this.handleDeclaration(decl);
+    public visitCallExpression(node: ts.CallExpression) {
```

Comment:
Hmm, I guess this actually bans the usage of any function named `require` ever, but I'm not sure there's any better way to do it.